backend/models/organizations.js

```diff
@@ -1,29 +1,22 @@
 const mongoose = require("mongoose");
 
 const user = new mongoose.Schema({
-  userId: {
-    type: String
-  },
-  admin: {
-    type: Boolean
+  id: {
+    type: mongoose.SchemaTypes.ObjectId,
+    unique: true
   }
 });
 
-//Currently we only have one role: reviewer.
-//In the future we may want to have "spectators" for example
 const program = new mongoose.Schema({
   id: {
-    type: String,
+    type: mongoose.SchemaTypes.ObjectId,
     unique: true
```

Comment:
I like the definition of `mongoose.SchemaTypes.ObjectId` instead of "String"

Reply:
Yep! I was thinking to update the other string references in these files too, but not sure what endpoints it'll break. I might update them later as I implement the endpoints.